### DISCUSSION TITLE
posts/632

### DISCUSSION THREAD

**Discussion by giscus** [kakaotech/comments#15] [Announcements]
# posts/632

안녕하세요, 데이터분석플랫폼 조직의 루이스입니다.
저희 조직의 주요 미션은 서비스...

https://tech.kakao.com/posts/632

**Comment by leonardBang** [comment]
Nice blog post even I don't know 한국인

**Reply by SML0127** [reply]
@leonardBang 
Thank you for reading my article and your kind comments. I will try to write in English next time😄😄

**Comment by milleryoon** [comment]
감사합니다. 잘 읽었습니다.

**Reply by SML0127** [reply]
@milleryoon 
긴 글 봐주셔서 정말 감사합니다😃😃😃

**Comment by benelog** [comment]
유익한 글 감사합니다!

**Reply by SML0127** [reply]
@benelog 
댓글 감사합니다. 댓글 달아 주시는 분들 덕분에 글 쓸때의 노고가 희미해지는 것 같습니다. 더 좋은 내용으로 다시 찾아뵐 수 있도록 정진하겠습니다.
